functions/Get-DbaTempdbUsage.ps1

```diff
@@ -0,0 +1,85 @@
+﻿Function Get-DbaTempdbUsage
+{
+    <#
+    .SYNOPSIS
+    Gets Tempdb usage for running queries.
+	
+    .DESCRIPTION
+    This function queries DMVs for running sessions using Tempdb and returns results if those sessions have user or internal space allocated or deallocated against them.
+	
+    .PARAMETER SqlInstance
+    The SQL Instance you are querying against.
+    .PARAMETER SqlCredential
+    If you want to use alternative credentials to connect to the server.
+    .PARAMETER Detailed
+    Returns additional information from the DMVs, such as:
+    -- program_name running the session.
+    -- login_time of the session.
+	
+    .PARAMETER WhatIf
+	Shows what would happen if the command were to run. No actions are actually performed.
+	.PARAMETER Confirm 
+	Prompts you for confirmation before executing any changing operations within the command.
+	
+	.PARAMETER Silent
+	Use this switch to disable any kind of verbose messages
+	
+    .NOTES
+    dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+    Copyright (C) 2016 Chrissy LeMaire
+    This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+    This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    .LINK
+    https://dbatools.io/Get-DbaTempdbUsage
+    .EXAMPLE
+    Get-DbaTempdbUsage -SqlInstance localhost\SQLDEV2K14
+	
+	Gets tempdb usage for localhost\SQLDEV2K14
+    .EXAMPLE
+    Get-DbaTempdbUsage -SqlInstance localhost\SQLDEV2K14 -Detailed
+	
+	Gets detailed tempdb usage for localhost\SQLDEV2K14
+    #>
+	[CmdletBinding()]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[object[]]$SqlInstance,
+		[object]$SqlCredential,
```

Comment:
Please, change this from [object] to [System.Management.Automation.PSCredential]